master/static/srv/tensorboard-entrypoint.sh

```diff
@@ -2,10 +2,8 @@
 
 source /run/determined/task-signal-handling.sh
 source /run/determined/task-logging-setup.sh
-trap 'source /run/determined/task-logging-teardown.sh' EXIT
 
 set -e
-set -x
```

Comment:
let's leave the `set -x` for now. or maybe @rb-determined-ai you were mentioning the logs, want to just put it around the startup script? I think that's acceptable. though it may be nice to do something like `if debug == true, set -x`.

Reply:
maybe just drop it. if users want to add it to their startup script they can. and then in a separate PR we can enable it when debug is on.

Reply:
I think it's a good idea to just drop it.  We're past the point of "so simple it's fine to show users every detail" now.